flask_monitoringdashboard/templates/fmd_dashboard/graph-details.html

```diff
@@ -1,71 +1,76 @@
+            {{ endpoint_submenu(url_version, 'Per-Version Performance') }}
+            {{ endpoint_submenu(url_user, 'Per-User Performance') }}
+            {{ endpoint_submenu(url_outliers, 'Outliers') }}
+        </ul>
+    </li>
 {% endblock %}
 
 {% block summary %}
-<div class="card mb-3" style="margin-top: 10px;">
-  <div class="card-header"><h4>Summary of endpoint</h4></div>
-  <div class="card-body">
-    <div class="table-responsive">
-      <table class="table table-striped table-bordered table-hover">
-        <tr>
-          <td><b>Endpoint</b></td>
-          <td>{{ details.endpoint }}</td>
-        </tr>
-        <tr>
-          <td><b>Added since app version</b></td>
-          <td>{{ details.rule.version_added }}</td>
-        </tr>
-        <tr>
-          <td><b>Date added to app</b></td>
-          <td>{{ "{:%Y-%m-%d %H:%M:%S }".format(details.rule.time_added) }}</td>
-        </tr>
-        {% if url %}
-          <tr>
-            <td><b>Link to endpoint</b></td>
-            <td><a href="{{ details.url }}">{{ details.url }}</a></td>
-          </tr>
-        {% endif %}
-        <tr>
-          <td><b>Last accessed</b></td>
-          <td>{{ "{:%Y-%m-%d %H:%M:%S }".format(details.rule.last_accessed) }}</td>
-        </tr>
-        <tr>
-          <td><b>Total number of hits</b></td>
-          <td>{{ "{:,d}".format(details.total_hits) }}</td>
-        </tr>
-      </table>
+    <div class="card mb-3" style="margin-top: 10px;">
+        <div class="card-header"><h4>Summary of endpoint</h4></div>
+        <div class="card-body">
+            <div class="table-responsive">
+                <table class="table table-striped table-bordered table-hover">
+                    <tr>
+                        <td><b>Endpoint</b></td>
+                        <td>{{ details.endpoint }}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Rule(s)</b></td>
+                        <td>{{ details.rules }}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Added since app version</b></td>
+                        <td>{{ details.rule.version_added }}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Date added to app</b></td>
+                        <td>{{ "{:%Y-%m-%d %H:%M:%S }".format(details.rule.time_added) }}</td>
+                    </tr>
+                    {% if url %}
+                        <tr>
+                            <td><b>Link to endpoint</b></td>
+                            <td><a href="{{ details.url }}">{{ details.url }}</a></td>
+                        </tr>
+                    {% endif %}
+                    <tr>
+                        <td><b>Last accessed</b></td>
+                        <td>{{ "{:%Y-%m-%d %H:%M:%S }".format(details.rule.last_accessed) }}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Total number of hits</b></td>
```

Comment:
all these changes seem to be simply code formatting changes, right? 
are you @FlyingBird95 @bogdanp05 @kloostert using the same IDE? 
do you have the same formatting style? 
otherwise we'll end up ping-ponging formatting changes all the time,
and they're confusing to review.

Reply:
Correct, except for the additional 'Rule(s)' row in the endpoint summary table. This is added to show which routes in the user app lead to the endpoint that is shown.
As far as I know, we're all using PyCharm. I reformatted the html, css and js files such that this should not lead to any formatting changes in the future anymore.